hw/ip/dma/data/dma.hjson

```diff
@@ -464,35 +464,49 @@
                 No explicit clearing necessary.
                 '''
         }
-        { bits: "5"
-          name: "memory_buffer_auto_increment_enable"
-          resval: 0x0
-          desc: '''
-                Used in conjunction with the hardware handshake mode of operation.
-                Auto Increments the memory buffer address register by data size to point to the next memory buffer address.
-                Generate a warning (assert interrupt) if the auto-incremented address reaches close to the value set in limit address register to prevent destination buffer overflow.
-                Enables firmware to take appropriate action prior to reaching the limit.
-                '''
-        }
-        { bits: "6"
-          name: "fifo_auto_increment_enable"
+        { bits: "6:5"
+          name: "src_addr_increment"
```

Comment:
These 'address increment' values sound like the numerical offset to be added to the src/dst address. Could they perhaps be renamed to 'src|dst_addr_mode' please? I think that's clearer.